### ISSUE TITLE
Still Needed?

### ISSUE THREAD

**Issue by Cessquill** [Xaero252/unraid-plex-nvdec#5]
This may sound odd, but am I right in thinking that this should no longer be run with the latest version of Plex?  Or will it still be OK?  Or are we in some kind of limbo whilst Unraid no longer has the current version of the driver installed for Plex?

Putting here, as the main Nvidia thread on the Unraid forum has requested posts like this be avoided.  However, also concerned that Unraid Nvidia might make Plex worse if latest drivers not installed.

**Comment by rmeaux** [comment]
Delete the script from User Scripts and restart Plex and let it roll. 

BTW, running the linuxserver container and it’s grabbing the latest Plex fine and it’s running smoothly.

**Comment by Cessquill** [comment]
Cool, cheers.  I hadn't run it since the latest batch of Plex releases, and all seemed OK (hadn't stress tested it though).  I'm just a little concerned about the minimum driver version that Plex stipulates, and the fact that it's newer than the Unraid version.

**Comment by NickBrecht** [comment]
Maybe I’m out of the loop but why wouldn’t the script be needed? Did nvidia remove the transcode limit or?

**Comment by Xaero252** [comment]
The newest versions of Plex have nvdec enabled on Linux. All this script
did was trick the older version of the Plex Transcoder into using the nvdec
option.

This has become obsolete as a result of official nvdec support.

**Comment by Xaero252** [comment]
Also, you may be confusing this with the transcode limit patch for the
Nvidia driver. I do not condone or support the use of that patch, as it
circumvents licensing restrictions Nvidia has put in place.

**Comment by Cessquill** [comment]
Yep, this script is not related to the Nvidia limit, it was to enable hardware decoding until Plex implemented it properly.
